dynamic-config.md

```diff
@@ -0,0 +1,216 @@
+---
```

Comment:
TOC.md in the upstream has been changed over Special Week. Please put this file link `dynamic-config.md` to TOC.md. @rleungx 

FYI @TomShawn

Reply:
@kissmydb PTAL